rand/rand.go

```diff
@@ -7,11 +7,6 @@ import (
 	"time"
 )
 
-func init() {
-	// initialize global pseudo random generator
```

Comment:
Could you explain why is it removed? Is `math.rand` seeded automatically now?

Btw, do we really need this package?

Reply:
From https://golang.org/doc/go1.20#math/rand :

```
The [math/rand](https://golang.org/pkg/math/rand/) package now automatically seeds the global random number generator (used by top-level functions like Float64 and Int) with a random value, and the top-level [Seed](https://golang.org/pkg/math/rand/#Seed) function has been deprecated. Programs that need a reproducible sequence of random numbers should prefer to allocate their own random source, using rand.New(rand.NewSource(seed)).
...
The top-level [Read](https://golang.org/pkg/math/rand/#Read) function has been deprecated. In almost all cases, [crypto/rand.Read](https://golang.org/pkg/crypto/rand/#Read) is more appropriate.
```

Reply:
I removed our own `rand` package. It was only used in a handful of places (only tests) and could easily be replaced with `math/rand` and `crypto/rand`